test/src/test/java/com/distkv/dst/test/jmhbenchmark/ListDstBenchmark.java

```diff
@@ -0,0 +1,83 @@
+package com.distkv.dst.test.jmhbenchmark;
+
+import com.distkv.dst.asyncclient.DefaultAsyncClient;
+import com.distkv.dst.asyncclient.DstAsyncClient;
+import com.distkv.dst.client.DefaultDstClient;
+import com.distkv.dst.client.DstClient;
+import com.distkv.dst.test.supplier.TestUtil;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang.RandomStringUtils;
+import org.openjdk.jmh.infra.Blackhole;
+import java.util.List;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Mode;
+
+@State(Scope.Thread)
+@BenchmarkMode({Mode.Throughput, Mode.AverageTime})
+public class ListDstBenchmark {
+
+  private DstAsyncClient asyncClient;
+  private DstClient client;
+  private List<String> dummyData;
+
+  @Setup
+  public void init() {
+    TestUtil.startRpcServer(8082);
+
+    dummyData = ImmutableList.of(
+        RandomStringUtils.random(5),
+        RandomStringUtils.random(5),
+        RandomStringUtils.random(5));
+
+    asyncClient = new DefaultAsyncClient("list://127.0.0.1:8082");
```

Comment:
Why the protocol is `list://`?

Reply:
And write `"list://127.0.0.1:8082"` as a static final

Reply:
I will new issue to fix it